index.php

```diff
@@ -45,16 +45,18 @@
 		<center>
 			<!-- banner class to establish what the site is about and what further content may entail (will be an image in later stages)-->
 			<div class="banner">
-				<h1 style="padding-top: 200px; font-size: 4.8vw;">Discover new music.</h1>
-				<h2>Tune in to your favourite songs by your favourite artists offline and ad-free today.</h2>
+				<h1 style="padding-top: 14vw; font-size: 4.8vw; color: white;">Discover new music.</h1>
+				<h2 style="margin-top: -1.5vw; font-size: 1.75vw;">Tune in to your favourite songs by your favourite artists today.</h2>
 			</div>
 
 			<!-- This class is for my main content-->	
 			<div class="content" style="height: 100%;">
-				<h1 style="text-align: left;">Your Library</h1>
+				<!-- All users 'saved playlists' (will be linked images in later stages)-->
+				<h1 style="color: white;">Your Library</h1>
 				<img style="width: 100%; max-width: 400px; max-height: 200px; padding: 25px;" src="images/placeholder.png" alt="Query 1">
 				<img style="width: 100%; max-width: 400px; max-height: 200px; padding: 25px;" src="images/placeholder.png" alt="Query 2"><br>
-				<h1 style="text-align: left;">Genres</h1>
```

Comment:
#removed left align for h1 headings to stick to consistency and standards and balance